www/front_src/src/Resources/api/index.ts

```diff
@@ -12,4 +17,58 @@ const listResources = (
 ): Promise<ResourceListing> =>
   getData({ endpoint: buildResourcesEndpoint(endpointParams), requestParams });
 
-export { listResources, getData };
+interface ResourceParams {
+  resource_id: string;
```

Comment:
we should create a column type number to manage those properties as integer
Furthermore, in the listing, a number column is generally aligned to the right